lib/IDE/ModuleInterfacePrinting.cpp

```diff
@@ -238,6 +238,295 @@ swift::ide::findGroupNameForUSR(ModuleDecl *M, StringRef USR) {
   return None;
 }
 
+/// Prints a single decl using the \p Printer and \p Options provided.
+///
+/// \returns Whether the given decl was printed.
+static bool printModuleInterfaceDecl(Decl *D,
```

Comment:
This isn't new code - it's just pulled out of `printSubmoduleInterface()` below.